examples/minimal_config_sync/main.tf

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Did you mean to commit this? We might want to leave out this example since it's very hard-coded.

Reply:
ugh, nope, thanks. dropped